build/NuGetPackages/Microsoft.Build.Framework.nuspec

```diff
@@ -17,13 +17,13 @@
       <group targetFramework=".NETStandard1.3">
         <dependency id="System.Threading.Tasks.Parallel" version="4.0.1" />
         <dependency id="System.Runtime.Extensions" version="4.1.0" />
+        <dependency id="System.Resources.ResourceManager" version="4.0.1" />
         <dependency id="System.Collections" version="4.0.11" />
         <dependency id="System.Diagnostics.Debug" version="4.0.11" />
         <dependency id="System.Diagnostics.Process" version="4.1.0" />
         <dependency id="System.Globalization" version="4.0.11" />
         <dependency id="System.Linq" version="4.1.0" />
         <dependency id="System.ObjectModel" version="4.0.12" />
-        <dependency id="System.Resources.ResourceManager" version="4.0.1" />
```

Comment:
Undo this change. For some reason the nuspec reference generator chooses a different ordering on the CI machines than on our local machines. :(

Reply:
Oops this was committed by accident